Chargebee/Classes/Configuration/Chargebee.swift

```diff
@@ -8,9 +8,11 @@ import Foundation
 public class Chargebee {
     public static let shared = Chargebee()
     private var client = NetworkClient()
+    public static var environment: String = "cb_ios_sdk"
```

Comment:
Does this need to be public?

Reply:
yes, its required to be public to set the environment in flutter.